examples/getstarted/config/admin.js

```diff
@@ -14,4 +14,5 @@ module.exports = ({ env }) => ({
       salt: env('TRANSFER_TOKEN_SALT', 'example-salt'),
     },
   },
+  isNpsEnabled: env('IS_NPS_ENABLED', true),
```

Comment:
I like the isSomething convention too but maybe we should follow the naming convention already established by other features such as audit logs. NPS_ENABLED. 

Similarly should we use env.bool here like audit logs?

Reply:
Both make sence want me to also remove the isNpsEnabled on strapi's end?